dataverse-webapp/src/main/webapp/editDatasetMetadata.xhtml

```diff
@@ -65,15 +66,14 @@
                     
                     <!-- Bottom action buttons -->
                     <div class="button-block">
-                        <p:commandButton tabindex="1000" id="save" value="#{bundle.saveChanges}" onclick="PF('datasetSaveButton').getJQ().click();" />
-                        <p:commandButton tabindex="1000" id="cancel" value="#{bundle.cancel}" action="#{editDatasetMetadataPage.cancel}" />
-                        <p:commandButton value="Direct" id="datasetSave"
+                        <p:commandButton value="#{bundle.saveChanges}"
                                          widgetVar="datasetSaveButton"
-                                         style="display:none"
-                                         update=":editMetadataForm,:messagePanel"
+                                         update=":editMetadataForm"
                                          onclick="PF('lockEditMetadataForm').show();"
                                          oncomplete="$(document).scrollTop(0);"
                                          action="#{editDatasetMetadataPage.save}" />
+                        <p:commandButton tabindex="1000" id="cancel" value="#{bundle.cancel}" action="#{editDatasetMetadataPage.cancel}" />
```

Comment:
Why this one has `tabindex` and the top one doesn't. And why do we set it to (effectively) infinity?

Reply:
I would have to check it out, the only reason I duplicated buttons is that when you use commandButton to click other button, there is double Ajax request so the error banner is disappearing in that case.

Reply:
I checked what it is, it is mainly used for Tabbing order, which setting it to 1k doesn't make sense IMO, so I removed it. But it is going to be important attribute for WCAG.